{
	"schema_version": "1.6.0",
	"id": "DLA-3039-1",
	"modified": "2023-06-28T06:26:48.243533Z",
	"published": "2022-06-03T00:00:00Z",
	"related": [
		"CVE-2022-24859"
	],
	"summary": "pypdf2 - security update",
	"details": "\nSebastian Krause discovered that manipulated inline images can force\nPyPDF2, a pure Python PDF library, into an infinite loop, if a maliciously\ncrafted PDF file is processed.\n\n\nFor Debian 9 stretch, this problem has been fixed in version\n1.26.0-2+deb9u1.\n\n\nWe recommend that you upgrade your pypdf2 packages.\n\n\nFor the detailed security status of pypdf2 please refer to\nits security tracker page at:\n<https://security-tracker.debian.org/tracker/pypdf2>\n\n\nFurther information about Debian LTS security advisories, how to apply\nthese updates to your system and frequently asked questions can be\nfound at: <https://wiki.debian.org/LTS>\n\n\n",
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:9",
				"name": "pypdf2",
				"purl": "pkg:deb/debian/pypdf2?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.26.0-2+deb9u1"
						}
					]
				}
			],
			"versions": [
				"1.26.0-2"
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/debian-osv/dla-osv/DLA-3039-1.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.debian.org/lts/security/2022/dla-3039"
		}
	]
}
